packages/unix_home.rb

```diff
@@ -0,0 +1,48 @@
+require 'package'
+
+class Unix_home < Package
+  description 'Regular Unix home directory for Chrome OS'
+  version '1.0'
+  compatibility 'all'
+  license 'GPL-3+'
+  source_url 'file:///dev/null'
+  source_sha256 'e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855'
+  
+  def self.install
+    @HOME = "#{CREW_DEST_PREFIX}/home/#{USER}/"
+    FileUtils.mkdir_p [@HOME, "#{CREW_DEST_PREFIX}/etc/env.d/"]
+    
+    FileUtils.ln_s Dir["#{Dir.home}/.bash*"], @HOME
```

Comment:
Isn't `#{Dir.home}` the same as `#{HOME}`?

Reply:
The `HOME` const will set to `CREW_PREFIX + Dir.home` if custom prefix set